test/upgrade_test/fission_objects.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+set -e
+
+ns="f-ns"
+ROOT=$(pwd)
+REPO="docker.io/library"
+STABLE_VERSION=1.12.0
+HELM_VARS="helmVars=repository=docker.io/library,image=fission-bundle,pullPolicy=IfNotPresent,imageTag=latest,fetcher.image=docker.io/library/fetcher,fetcher.imageTag=latest,postInstallReportImage=reporter,preUpgradeChecksImage=preupgradechecks" 
+
+getVersion () {
+    echo $(git rev-parse HEAD)
+}
+
+getDate () {
+    echo $(date -u +'%Y-%m-%dT%H:%M:%SZ')
+}
+
+getGitCommit () {
+    echo $(git rev-parse HEAD)
+}
+
+getNameSpace () {
+    ns = $(cat ./namespace.txt)
+}
+
+dump_system_info () {
+    echo "System Info"
+    go version
+    docker version
+    kubectl version
+    helm version
+}
+
+install_stable_release () {
+    echo "Creating namespace $ns"
+    kubectl create ns $ns
+    echo "Installing Fission $STABLE_VERSION"
+    helm install \
+    --namespace $ns \
+    --name-template fission \
+    https://github.com/fission/fission/releases/download/${STABLE_VERSION}/fission-all-${STABLE_VERSION}.tgz
+    mkdir temp && cd temp && curl -Lo fission https://github.com/fission/fission/releases/download/${STABLE_VERSION}/fission-cli-linux && chmod +x fission && sudo mv fission /usr/local/bin/ && cd .. && rm -rf temp
+    sleep 10
```

Comment:
Why do we need sleep here? If not needed we should remove it.

Please add comments/reason in code wherever we are using `sleep`.

Reply:
This sleep is required here to become all pods active. Have added comments  in script.